r2snowman/r2snow

```diff
@@ -1,18 +1,15 @@
-#!/bin/sh
-TARGET="$1"
+#!/usr/bin/python
+import r2pipe
+from pygments import highlight
+from pygments.lexers import CppLexer
+from pygments.formatters import Terminal256Formatter
+from os import popen
 
-if [ -z "$1" ]; then
```

Comment:
why remove the help? you are removing the ability to use r2snow from the commandline, check for this if r2pipe.open() fails